arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -3669,6 +3679,31 @@
 			path = "Distance composite symbol";
 			sourceTree = "<group>";
 		};
+		F151AB8A22D7B07800055E45 /* Monterey_Elevation */ = {
+			isa = PBXGroup;
+			children = (
+				F151AB8B22D7B0CA00055E45 /* MontereyElevation.dt2 */,
+			);
+			path = Monterey_Elevation;
+			sourceTree = "<group>";
+		};
+		F1679F0422D93FA900E5320A /* Rasters */ = {
```

Comment:
I'm not very familiar with how content is named/organized within these samples, do you think it makes sense for the comment to be something more descriptive, instead of` /* Rasters */` I'm ok either way, just wanted to point it out

Reply:
I'm not sure what you mean by "comment." This change comes from a group that I added under `/Portal Data` to store the raster file.